website/content/docs/concepts/policies.mdx

```diff
@@ -798,34 +798,45 @@ authenticated user.
-| [sys/leases/revoke-prefix/:prefix](/vault/api-docs/system/leases#revoke-prefix)                                                                        | PUT               | Revoke all secrets generated under a given prefix                                                                                                   |
-| [sys/plugins/catalog/:type/:name](/vault/api-docs/system/plugins-catalog#register-plugin)                                                              | GET, PUT, DELETE  | Register a new plugin, or read/remove an existing plugin                                                                                            |
-| [sys/raw](/vault/api-docs/system/raw#list-raw)                                                                                                         | LIST, GET         | Returns a list of keys for a given path prefix                                                                                                      |
-| [sys/replication/reindex](/vault/api-docs/system/replication#reindex-replication)                                                                          | POST              | Reindex the local data storage                                                                                                                      |
+| [sys/leases/revoke-force/:prefix](/vault/api-docs/system/leases#revoke-force)                                                                          | POST              | Revoke all secrets or tokens ignoring backend errors                                                                                                |
+| [sys/leases/revoke-prefix/:prefix](/vault/api-docs/system/leases#revoke-prefix)                                                                        | POST              | Revoke all secrets generated under a given prefix                                                                                                   |
+| [sys/plugins/catalog/:type/:name](/vault/api-docs/system/plugins-catalog#register-plugin)                                                              | GET, POST, DELETE | Register a new plugin, or read/remove an existing plugin                                                                                            |
+| [sys/raw:path](/vault/api-docs/system/raw)                                                                                                             | GET, POST, DELETE | Used to access the raw underlying store in Vault                                                                                                    |
+| [sys/raw:prefix](/vault/api-docs/system/raw#list-raw)                                                                                                  | GET, LIST         | Returns a list keys for a given path prefix                                                                                                         |
```

Comment:
This new representation of `sys/raw` does not communicate well to users.

There is a single API, which can be best described as `sys/raw/:path`, accepting GET, LIST, POST, DELETE:

```suggestion
| [sys/raw/:path](/vault/api-docs/system/raw)                                                                                                             | GET, LIST, POST, DELETE | Used to access the raw underlying store in Vault                                                                                                    |
```

It doesn't make sense to single out LIST to a separate row, use different path-parameter placeholders for the two rows, duplicate GET over both of them, or omit the slash character before the colons.

Reply:
Thanks for checking in. I match the table to our docs which list `/sys/raw/:prefix` and `sys/raw:path` on https://developer.hashicorp.com/vault/api-docs/system/raw